resources/function_help/json/overlay_intersects

```diff
@@ -38,6 +38,16 @@
       "arg": "min_inscribed_circle_radius",
       "description": "defines an optional exclusion filter (for polygons only): minimum radius in current feature units for the maximum inscribed circle of the intersection (if the intersection results in multiple polygons the intersection will be returned if at least one of the polygons has a radius for the maximum inscribed circle greater or equal to the value).<br>Read more on the underlying GEOS predicate, as described in PostGIS <a href='https://postgis.net/docs/ST_MaximumInscribedCircle.html'>ST_MaximumInscribedCircle</a> function.<br>This argument requires GEOS >= 3.9.",
       "optional": true
+    },
+    {
+      "arg": "return_measure",
+      "description": "only valid when used with an expression, set this to true to return a list of maps containing the feature id, the expression result and the overlap value, the radius of the maximum inscribed circle is also returned when the target layer is a polygon.",
+      "optional": true
+    },
+    {
+      "arg": "sort_by_intersection_size",
```

Comment:
If a user wants an ascending order of results, I guess the only option is to array_reverse the function's output. There's no `false`, right? What do you think about having `ascending`/`descending` as values for this parameter instead of `true`?

Reply:
@DelazJ - do you think there is a use case for ascending order?

These improvements are mostly for getting only the largest intersection object, or the 2 or 3 largest ones.

Reply:
> do you think there is a use case for ascending order?

*playing the devil* someone looking for digitizing glitches? Find that target feature which overlaps my feature while it shouldn't have?
But TBH @andreasneumann , I didn't think about the use case when suggesting. It's the parameter name that made me think that I could sort in both directions, before I read the description.

Reply:
I think allowing ascending order is fair, even if less likely to be used.

Reply:
There is no technical difficulty in adding sort direction, I'd just like to hear a use case for that: when would it be useful to have smaller overlaps listed first? Remember you still have a `min_overlap` that filters out all overlaps bigger than the value.

Reply:
I think being able to identify the smallest feature within a bigger one could be helpful if someone wants to find things such as the smallest habitat/community/etc within a bigger area. To me wanting the biggest or the smallest unit within another one are both equally reasonable to request. Either for protection, for removal or to clear up broken geometries even. Let's say you have a sliver in a dataset and you don't want to look for it manually, setting a symbology would help find those small geometries and remove them until the data is 'cleaned'.
 Though wanting a use-case may be convincing, I don't think they are needed at creating time and from a small subset of users (mostly dev) to validate if something will be useful or not to the larger user-base.

Reply:
ok, done.